.github/workflows/docker.yml

```diff
@@ -15,4 +16,4 @@ jobs:
         run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
       - name: Push image
         run: |
-          ./build-and-push.sh
+          export COUNTRY_CONFIG_VERSION=`git log -1 --pretty=format:%h` && bash build-and-push.sh && unset COUNTRY_CONFIG_VERSION
```

Comment:
Notice that we also now got [this](https://github.com/opencrvs/opencrvs-zambia/blob/develop/.github/workflows/publish-to-dockerhub.yml) pipeline. Could you move this edit to that file and remove this pipeline?